packages/website/docs/MultiSelect.md

```diff
@@ -1,23 +1,50 @@
-## Guidelines
+## Best Practices
 
-### General
+Use multi-select to allow user to select options from a **long list that can be filtered**. Multi-selects are more appropriate when the available list of options is very long or when the space is limited.
 
--   If space allows, and there are no more than 7 options in total, consider using [checkboxes](#/components/Checkbox) instead.
--   Keep option labels as short as possible in order to limit width to a minimum.
--   When an option is selected, it should be removed from the options list.
+Use alphanumerical order unless a more suited ordering rational applies (such as sizes or security levels).
 
-### Options Sorting
+Always include the ability to filter the list when there are over 20 options.
 
--   Always put the most common items at the top of the list. If no option is more common than the others, use alphanumeric or chronological order.
+## Labeling
 
-### Label
+Keep titles and labels short, **under one line**.
```

Comment:
In the two other PRs I reviewed, we had:
```suggestion
Keep titles and labels short, preferably **under three words**.
```
I don't know if we want to say the same here? At the top of my head, I can't see how this is different from the other components?